app/builders/messages/instagram/direct/message_builder.rb

```diff
@@ -0,0 +1,42 @@
+class Messages::Instagram::Direct::MessageBuilder < Messages::Instagram::BaseMessageBuilder
+  include HTTParty
```

Comment:
might not need this.

Reply:
Used [here](https://github.com/chatwoot/chatwoot/pull/11054/files#diff-afe31ea079d16339feca89d76347f8c2a9b221edb64a5080a01aa54d4b04e151R14)